dex/src/main/scala/com/wavesplatform/dex/api/websockets/WsOrder.scala

```diff
@@ -26,6 +26,16 @@ case class WsOrder(id: Order.Id,
 
 object WsOrder {
 
+  def orderDiff(ao: AcceptedOrder, status: Option[String], fa: Double, ff: Double, awp: Double): WsOrder = {
+    WsOrder(
+      ao.id,
+      status = status,
+      filledAmount = Some(fa),
+      filledFee = Some(ff),
+      avgWeighedPrice = Some(awp)
+    )
+  }
```

Comment:
Do you use this whenever its possible?

Reply:
I used it one time, but maybe it can be useful in the future time

Reply:
I find a way to remove `orderDiff`